src/pymodaq/utils/leco/daq_move_LECODirector.py

```diff
@@ -148,12 +149,12 @@ def _set_position_value(
     ) -> DataActuator:
         if position:
             if isinstance(position, str):
-                deserializer = DeSerializer.from_b64_string(position)
-                pos = deserializer.dwa_deserialization()
+                decoded = b64decode(position)
```

Comment:
could you recall why you're using a b64decode and not plain encoder?

Reply:
Originally, we could not transfer binary data, just JSON, therefore I went to use that encoding/decoding.
In this PR, I do not encode via base64 anymore, but for dependency, I decode.

If you want to restrict PyLECO communication to PyMoDAQ 5 among itself, we can remove all the b64 decoding. (see #469 )